ral-facilities/operationsgateway
# DSEGOG-227 Changes to datapicking and timeframes

## Description
Remove the consideration of seconds when picking dates and setting timeframes.
Set the seconds of the from date to 00 and the seconds of the to date to 59.
Disable the selection of seconds in the datepicker.

## Testing instructions
- Verify that the timeframes are set correctly by ensuring that the from date has its seconds set to 00 and the to date has its seconds set to 59.
- Confirm that the datepicker cannot select seconds and that the from and to dates have their seconds set to 00 and 59, respectively. 


## Agile board tracking
Closes DSEGOG-227
